mappings/net/minecraft/entity/WaterCreatureEntity.mapping

```diff
@@ -1,5 +1,5 @@
 CLASS net/minecraft/class_1480 net/minecraft/entity/WaterCreatureEntity
 	METHOD <init> (Lnet/minecraft/class_1299;Lnet/minecraft/class_1937;)V
 		ARG 1 type
-	METHOD method_6673 tickBreath (I)V
-		ARG 1 breath
+	METHOD method_6673 tickAir (I)V
```

Comment:
I think this name can be improved. This method is specifically for fish and creatures that cannot breathe outside water, and is made no-op in dolphins.

Reply:
I renamed it to `tickWaterBreathingAir`, but I'm not sure if that's the best mapping for the method.